leetcode/src/236.c

```diff
@@ -0,0 +1,81 @@
+/**
+ * Definition for a binary tree node.
+ * struct TreeNode {
+ *     int val;
+ *     struct TreeNode *left;
+ *     struct TreeNode *right;
+ * };
+ */
+
+// The list for TreeNodes.
+struct ListItem {
+    struct TreeNode* node;
+    struct ListItem* next;

```

Comment:
Please add a one-line description of what the variable is for (**see the example below**).

```c
int64_t data = 0;       ///< The value/key of the node.
```

Reply:
fixed